src/layouts/BaseLayout.js

```diff
@@ -201,8 +201,10 @@ const breadcrumbNameMap = {
     title: 'List of available warehouses',
   },
   /* Stakeholders Routes */
```

Comment:
So the navigation to agencies won't be available here

Reply:
Okay, I have fixed it